packages/world/src/AccessControl.sol

```diff
@@ -44,7 +44,7 @@ library AccessControl {
   ) internal view returns (bytes32 resourceSelector) {
     resourceSelector = ResourceSelector.from(namespace, file);
 
-    if (NamespaceOwner.get(namespace) != caller) {
+    if (NamespaceOwner.get(namespace) != caller && caller != address(this)) {
```

Comment:
tiny thing but this might be more readable if `caller` is in the same position for each conditional check, also might be a little more gas efficient if the `address(this)` check is first
```suggestion
    if (caller != address(this) && caller != NamespaceOwner.get(namespace)) {
```